### ISSUE TITLE
Support graphing

### ISSUE THREAD

**Issue by rodya-mirov** [rodya-mirov/longleaf#7]
We want to graph functions. Ideally this would come in two flavors:

- A window popping up, displaying the graph, and
- An image file is created, which contains the graph

💅 can be done as a separate issue, this just calls for making graphs at all. Probably blocked by #6 and some other things. Needs design.

Intending to use luminance https://crates.io/crates/luminance for this but I'm willing to be convinced otherwise.
